plotting/sound.py

```
@@ -56,40 +56,162 @@ def plot(self):
+
+        # Render point location
+        if self.showmap:
+            plt.subplot(gs[0, 0])
+            utils.point_plot(np.array([[x[0] for x in self.points],  # Latitudes
+                                       [x[1] for x in self.points]]),
+                                        colors=colors)  # Longitudes
+
 
         minspeed = np.amin(self.sspeed)
         maxspeed = np.amax(self.sspeed)
+        print(something)
```

Ruff (0.12.2)

Undefined name `something`

(F821)